Dockerfile

```diff
@@ -64,7 +64,7 @@ else
   )
 fi
 find ${TARGET} -type f -name '*.html' | while read i; do
-  sed -i 's#\(<a[^>]* href="\)https://${DOMAIN}/#\1/#g' "$i"
+  sed -i "s#\(<a[^>]* href=\"\)https://${DOMAIN}/#\1/#g" "$i"
```

Comment:
Have we checked if the existing escapes need to be double-escaped now? (e.g. `\)` to be `\\)`) (escaping and quoting and shells are hard!)

Reply:
Looks good:

```console
$ DOMAIN=https://docs.docker.com
$ sed "s#\(<a[^>]* href=\"\)${DOMAIN}/#\1/#g" <<END
<a href="https://docs.docker.com/desktop/" id="manuals">Manuals</a>
END
<a href="/desktop/" id="manuals">Manuals</a>
```

Reply:
Thanks! Sorry, probably should've tried locally 😂 I know I ran into `sed` + regex + escaping fun in the past, so thought; let me leave a quick comment

Reply:
> I know I ran into sed + regex + escaping fun in the past,

Yes me too!